cmd/clusterctl/api/v1alpha3/metadata_type.go

```diff
@@ -32,29 +32,29 @@ type Metadata struct {
 	ReleaseSeries []ReleaseSeries `json:"releaseSeries"`
 }
 
-// ReleaseSeries maps a provider release series (major/minor) with a ClusterAPIVersion
+// ReleaseSeries maps a provider release series (major/minor) with a API Version of Cluster API (contract).
 type ReleaseSeries struct {
 	// Major version of the release series
 	Major uint `json:"major,omitempty"`
 
 	// Minor version of the release series
 	Minor uint `json:"minor,omitempty"`
 
-	// ClusterAPIVersion indicates the cluster API supported version.
-	ClusterAPIVersion string `json:"clusterAPIVersion,omitempty"`
+	// Contract defines the API Version of Cluster API (contract) supported by the ReleaseSeries.
+	Contract string `json:"contract,omitempty"`
 }
 
 func init() {
 	SchemeBuilder.Register(&Metadata{})
 }
 
-// HasReleaseSeriesForVersion return true if the given version is in one of the ReleaseSeries.
-func (m *Metadata) HasReleaseSeriesForVersion(version *version.Version) bool {
+// GetReleaseSeriesForVersion return the release series for a given version.
+func (m *Metadata) GetReleaseSeriesForVersion(version *version.Version) *ReleaseSeries {
```

Comment:
Switching from `Has` to `Get` so I can reuse the same function in `upgrade apply`